.travis.yml

```diff
@@ -22,34 +22,46 @@ branches:
     - /^test-.*$/
 
 env:
-    - RUN="lints integration generate rpm" BOULDER_CONFIG_DIR="test/config"
+    # With no options passed: runs lint, unit, and integation
+    # l: Adds lint to the list of tests to run
+    # u: Adds unit to the list of tests to run
+    # d: Run unit tests for a specific directory
+    # e: Enables -race flag for all unit tests
+    # n: Changes BOULDER_CONFIG_DIR from test/config to test/config-next
+    # c: Adds coverage to the list of tests to run
+    # i: Adds integration to the list of tests to run
+    # s: Adds start (py) to the list of tests to run
+    # v: Adds gomod-vendor to the list of tests to run
+    # g: Adds generate to the list of tests to run
+    # r: Adds rpm to the list of tests to run
+    # p: Outputs a list of the available integration tests
+    # f: Run only those tests and examples matching the regular expression
+    - TESTFLAGS="ligr"
```

Comment:
This documentation winds up being redundant with the comments in `test.sh` and is likely to get out of date. I would prefer to remove the comment here, and use the long form of the options in `TESTFLAGS`. One of the important uses of `TESTFLAGS` is, when looking at a list of tasks in which some failed, figuring out which ones failed; so it's useful to see the long form there.

Reply:
>This documentation winds up being redundant with the comments in test.sh and is likely to get out of date. 

I completely agree that this runs the risk of not being kept up-to-date. I'll remove it.

>I would prefer to remove the comment here, and use the long form of the options in TESTFLAGS

I used the shortopt flags here almost explicitly because they are far less likely to change than the longopt flags. Using longopts here will result in more edits to the `.travis.yml`, meaning more risk that we accidentally disable a test

>One of the important uses of TESTFLAGS is, when looking at a list of tasks in which some failed, figuring out which ones failed

Unless I am missing something, this is a role better filled by the another new feature added in this PR; the full settings message are output in bold blue before each test run. And as each test section commences, the name of that test section is out, with a newline, in bold blue:

```
Boulder Test Suite CLI

Settings:
    RUN:                integration unit
    BOULDER_CONFIG_DIR: test/config
    UNIT_FILTER:        -p 1 ./...
    ENABLE_UNIT_RACE:   false
    INT_FILTER:         --filter TestAkamaiPurgerDrainQueueFails

Starting...

Running Unit Tests
...unit output...

Running Integration Tests
...test output...
```

Reply:
I'm talking about this page: https://github.com/letsencrypt/boulder/pull/5155/checks?check_run_id=1361309571

![image](https://user-images.githubusercontent.com/220205/98425499-32fc5b00-204a-11eb-9613-950a26184bdf.png)

It's nice to be able to see which phases failed before clicking through. For instance, was it both integration test phases? Or one unittest phase in config-next?

Reply:
Great point. This is pretty hard to read in it's current form.